docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -756,15 +756,15 @@ WeakReferenceMessenger.Default.Reset();
 // MvvmLight
 IMessenger messenger = Messenger.Default;
 
-// Toolkit.Mvvm
+// MVVM Toolkit
 IMessenger messenger = WeakReferenceMessenger.Default;
 ```
 
 ## Migrating message types
 
 The message types provided in the MvvmLight toolkit are designed as a base for you as a developer to work with if needed.
 
-While the MVVM Toolkit provides some alternatives, there are no direct replacement for these message types. We recommend looking at our [available message types](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/rel/7.1.0/Microsoft.Toolkit.Mvvm/Messaging/Messages).
+While the MVVM Toolkit provides some alternatives, there are no direct replacement for these message types. We recommend looking at our [available message types](https://github.com/CommunityToolkit/dotnet/tree/main/src/CommunityToolkit.Mvvm/Messaging/Messages).
```

Comment:
Should probably still use a fixed commit path here as if we move stuff later in the repo (however unlikely) these links break.